lib/virtus/attribute/nullify_blank.rb

```diff
@@ -0,0 +1,24 @@
+module Virtus
+  class Attribute
+
+    # Attribute extension which nullifies blank attributes when coercion failed
+    #
+    module NullifyBlank
+
+      # @see [Attribute#coerce]
+      #
+      # @api public
+      def coerce(input)
+        output = super
+
+        if !value_coerced?(output) && input.to_s.empty?
+          nil
+        else
+          output
+        end
+      end
```

Comment:
I wonder if that could be collapsed to:

``` ruby
input == '' ? nil : super
```

Since we're only interested whether or not it's an empty string I guess this would suffice hmm